docs/learning/workshop_a_precision_converter_fpga_integration_journey/index.rst

```diff
@@ -0,0 +1,693 @@
+A Precision Converter FPGA Integration Journey
+==============================================
+
+Agenda
+~~~~~~
+
+1. Introduction
+2. SPI Engine Architecture
+3. What Is the Use Case?
+4. System Build
+5. System Evaluation
+6. Conclusions
```

Comment:
Should do automatic numbering, with #. and not manually write the numbers

Reply:
Done